assets/css/style.css

```diff
@@ -160,3 +228,67 @@ body {
   font-weight: 600;
 }
 
+.my_works {
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+
+.my_works h2 {
+  color: black;
+
+  font-size: 20px;
+  line-height: 1.1;
+  font-weight: 700;
+
+  text-transform: uppercase;
+  letter-spacing: 2px;
+}
+
+.my_work_enzym {
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+
+.my_work_enzym h3 {
+  font-size: 12px;
+  font-weight: 700;
+
+  text-transform: uppercase;
+  letter-spacing: 2px;
```

Comment:
font-size, text-transform, letter-spacing
this 3 properties are used minimum 3 times, try to DRY